benchmark/wikisql/wiki_sql.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from argparse import ArgumentParser
+
+from babel.numbers import parse_decimal, NumberFormatError
+from pyspark.sql import SparkSession
+
+from pyspark_ai import SparkAI
+
+
+def generate_sql_statements(table_file):
```

Comment:
nit: Shall we add a docstring to call out the sqls generated are to represent tables as views?

Reply:
Sure, updated